staging/src/kubevirt.io/client-go/api/v1/types.go

```diff
@@ -983,6 +983,8 @@ const (
 	CacheNone DriverCache = "none"
 	// CacheWriteThrough - I/O from the guest is cached on the host but written through to the physical medium.
 	CacheWriteThrough DriverCache = "writethrough"
+	// CacheWriteBack - I/O from the guest is cached on the host.
+	CacheWriteBack DriverCache = "writeback"
```

Comment:
If we add this, we'll need to make sure to document the potential data integrity trade off that is occurring with this cache mode.

Reply:
+ There is a good summary in https://docs.openstack.org/ocata/config-reference/compute/config-options.html